src/main.py

```diff
@@ -0,0 +1,345 @@
+from fastapi import FastAPI, Request, Form, HTTPException
+from fastapi.responses import HTMLResponse, RedirectResponse
+from fastapi.staticfiles import StaticFiles
+from fastapi.templating import Jinja2Templates
+from fastapi.encoders import jsonable_encoder
+
+from typing import List, Optional
+
+from db.models import User, Author, Venue, Keyword, Article, Reference, ArticleAuthor, ArticleKeyword
+from db.db_params import get_session
+from db.pmodels import PUserCreate, PUser, PAuthorCreate, PAuthor, PArticleCreate, PArticle, PVenueCreate, PVenue, \
+    PKeywordCreate, PKeyword, PArticleAuthor, PArticleKeyword, PReference
+
+app = FastAPI()
+
+app.mount("/static", StaticFiles(directory="static"), name="static")
+
+templates = Jinja2Templates(directory="templates")
+
+session = next(get_session())
+
+@app.get("/")
+def hello_world():
+    return {"message": "Hello World"}
+
+@app.get("/login", include_in_schema=False, response_class=HTMLResponse)
+def UI_login_get(request: Request):
+    return templates.TemplateResponse("login.html", {"request": request})
+
+@app.post("/login", include_in_schema=False, response_class=HTMLResponse)
+def UI_login_post(username: str, password: str):
+    response = RedirectResponse('main')
+    response.set_cookie(key="username", value=username)
+    return response
+
+### USERS
+
+@app.get("/users/", tags=["Users"], response_model=List[PUser])
+def users_get():
+    return session.query(User).all()
```

Comment:
See [this comment](https://github.com/TheoLisin/recsys_MADE/pull/3/files/85c9b31ca13bf17ca018c5ebf4457883c7657f84..fb998a9c1f7ce00f552994288ba98544f838fb19#r986221784)
```suggestion
    with get_session() as session:
        return session.query(User).all()
```